worldedit-core/src/main/java/com/fastasyncworldedit/core/FaweAPI.java

```diff
@@ -5,11 +5,11 @@
 import com.fastasyncworldedit.core.beta.implementation.lighting.Relighter;
 import com.fastasyncworldedit.core.beta.implementation.queue.ParallelQueueExtent;
 import com.fastasyncworldedit.core.configuration.Settings;
-import com.fastasyncworldedit.core.object.RegionWrapper;
-import com.fastasyncworldedit.core.object.RelightMode;
-import com.fastasyncworldedit.core.object.changeset.DiskStorageHistory;
-import com.fastasyncworldedit.core.object.changeset.SimpleChangeSetSummary;
-import com.fastasyncworldedit.core.object.exception.FaweException;
+import com.fastasyncworldedit.core.regions.RegionWrapper;
+import com.fastasyncworldedit.core.beta.implementation.lighting.RelightMode;
+import com.fastasyncworldedit.core.function.operation.changeset.DiskStorageHistory;
+import com.fastasyncworldedit.core.function.operation.changeset.SimpleChangeSetSummary;
```

Comment:
WorldEdit uses history.changeset rather than functyion. history makes much more sense than function as this is not a function as well.